src/main/webapp/js/feedbackSubmissionsEdit.js

```diff
@@ -447,7 +447,7 @@ function formatRecipientLists(){
             find("option[value='"+lastSelectedOption+"']").show();
         }
         if (curSelectedOption != "") {
-            $("select[name|="+FEEDBACK_RESPONSE_RECIPIENT+"-"+questionNumber+"]").not(this).
+        	$("select[name|="+FEEDBACK_RESPONSE_RECIPIENT+"-"+questionNumber+"]").not(this).
```

Comment:
I accidently replaced 4 spaces by a tab. I will change it back when adding test case for this issue.